### PR TITLE
index.md translation in french

### PR DESCRIPTION
**Pull request by justicengom**
I translated the index.md in french docs/basic_training/index-fr.md

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

**Comment by mribeirodantas** [comment]
Salut, @justicengom ! Very happy to see we'll have a French version of the training material 🥰

If you need help or have questions about the translation process, please, ask! 😃
You should first check the translating guidelines [here](https://github.com/nextflow-io/training/blob/master/TRANSLATING.md). Two things are specially important:

1. As this is the first translation to French, you need to change the `mkdocs.yml` file following [this example](https://github.com/nextflow-io/training/pull/163/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2), when Spanish was added.
2. Translated files should follow a dot notation. Your index file should be named `index.fr.md` instead of the current `index-fr.md`.

It seems you're about to start working on the translation of the `setup.md` file too. If this is the case, convert this Pull Request to a draft and make it ready for review when you're done, so that we know it's ready for review and we can approve/request changes/mege 🥳 
Let me know if you have questions.

**Comment by justicengom** [comment]
Thank you so much @mribeirodantas. I have implemented the changing

File: docs/basic_training/index.fr.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+description: Aperçu du matériel de formation Nextflow de base
+cacher :
```
Comment (mribeirodantas) [inline docs/basic_training/index.fr.md:3]:
```suggestion
hide:
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]
Great, @justicengom! There's only a minor thing missing :smiley

File: docs/basic_training/index.fr.md (hunk 1 above)
Comment (justicengom) [reply docs/basic_training/index.fr.md]: Thanks again @mribeirodantas , I've accepted the suggestion.

**Review by mribeirodantas — APPROVED** [review]

**Pull request merged**